checker/src/main/java/org/checkerframework/checker/guieffect/GuiEffectTypeFactory.java

```diff
@@ -548,7 +519,7 @@ public AnnotatedTypeMirror getAnnotatedType(Tree tree) {
         // We don't need to issue warnings for inheriting from poly and a concrete effect.
```

Comment:
Should "inheriting from" be "overriding"?

Reply:
I've fixed this bit.